swo/mpt/cli/core/errors.py

```diff
@@ -58,7 +58,7 @@ class NoActiveAccountFoundError(CLIError):
     def __str__(self) -> str:
         return (
             "No active account found. Activate any account first using "
-            "'swocli accounts active ACCOUNT-ID' command"
+            "'swocli accounts activate ACCOUNT-ID' command"
```

Comment:
We should change swocli here for "progname" in some way. The cli command has been renamed to mpt-cli

Reply:
@ivan-gurin since you are touching this place, can you please rename also swocli to mpt-cli? We'll implement Francesco's suggestion later :-)

Reply:
that actually did come across my mind from the start, but I couldn't find any evidence of such practice in existing code